components/centraldashboard/public/components/main-page.js

```diff
@@ -303,6 +303,15 @@ export class MainPage extends utilitiesMixin(PolymerElement) {
     _namespaceChanged(namespace) {
         // update namespaced menu item when namespace is changed
         // by namespace selector
+
+        if (namespace) {
+            // Save the user's choice so we are able to restore it, 
+            // when re-loading the page without a queryParam
+            const localStorageKey = "selectedNamespace" + 
+                (this.user && "." + this.user || "");
```

Comment:
Could we change this so that the localStorage key is `/centraldashboard/selectedNamespace/<user>`?

I'd like to make it explicit which localStorage key is set by which app, in case in the future more apps would use the localStorage